deploy/olm-catalog/opendatahub/1.2.0/manifests/opendatahub-operator.v1.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,348 @@
+                  }
+                },
+                "name": "prometheus-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "prometheus/operator"
+                  }
+                },
+                "name": "prometheus-operator"
+              },
+              {
+                "kustomizeConfig": {
+                  "parameters": [
+                    {
+                      "name": "s3_endpoint_url",
+                      "value": "s3.odh.com"
+                    }
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "jupyterhub/jupyterhub"
+                  }
+                },
+                "name": "jupyterhub"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "additional"
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "jupyterhub/notebook-images"
+                  }
+                },
+                "name": "notebook-images"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "airflow/operator"
+                  }
+                },
+                "name": "airflow-operator"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "airflow/cluster"
+                  }
+                },
+                "name": "airflow-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odhargo/cluster"
+                  }
+                },
+                "name": "odhargo-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odhargo/odhargo"
+                  }
+                },
+                "name": "odhargo"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odh-dashboard"
+                  }
+                },
+                "name": "odh-dashboard"
+              }
+            ],
+            "repos": [
+              {
+                "name": "kf-manifests",
+                "uri": "https://github.com/opendatahub-io/manifests/tarball/v1.4.0-rc.2-openshift"
+              },
+              {
+                "name": "manifests",
+                "uri": "https://github.com/opendatahub-io/odh-manifests/tarball/v1.2.0"
```

Comment:
This needs to be updated to v1.2 , if we are changing the tag in `odh-manifests` repo

Reply:
Good catch.  The `v1.2.0` tag works but using the `v1.2` branch will allow us to update/fix `v1.2` manifests without requiring an operator update